util/Migrator/DbScripts/2021-11-02_00_OrganizationSponsorship.sql

```diff
@@ -120,7 +120,7 @@ CREATE PROCEDURE [dbo].[OrganizationSponsorship_Create]
     @FriendlyName NVARCHAR(256),
     @OfferedToEmail NVARCHAR(256),
     @PlanSponsorshipType TINYINT,
-    @ToDelete BIT NULL,
+    @CloudSponsor BIT NULL,
```

Comment:
Still want this as `toDelete` and delete `cloudSponsor`

Reply:
This was not meant to be changed, I must have had it open and changed it recently. I've changed it back.